### PR TITLE
feat: update callback addresses for production use

### PR DESCRIPTION
**Pull request by zodahu**

### CODE REVIEW

File: src/logics/aave-v3/configs.ts
Comment (Jeff-CCH) [inline src/logics/aave-v3/configs.ts:1]: Does Gnosis Chain need to be included?

File: .changeset/twelve-cobras-remember.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@protocolink/logics': patch
+---
+
+fix: remove stable rate tests
```
Comment (Jeff-CCH) [inline .changeset/twelve-cobras-remember.md:5]: Wondering which one is better: removing or skipping?

Comment (Jeff-CCH) [inline .changeset/twelve-cobras-remember.md:5]:
```suggestion
fix: remove AAVE V2 stable rate tests
```

Comment (Jeff-CCH) [reply .changeset/twelve-cobras-remember.md:5]: Seems AAVE will not reopen the stable borrow in the near future. We will remove the tests.

**Review by chouandy — APPROVED** [review]
LGTM

**Pull request merged**